GitCommands/Settings/AppSettings.cs

```diff
@@ -2108,6 +2108,15 @@ void AddEncodingByName(string s)
                 {
                     // there are CultureInfo values without a code page
                 }
+
+                try
+                {
+                    AddEncoding(Encoding.GetEncoding(0));
```

Comment:
Does AddEncodings prevent duplicate encodings?  If default is one that is also added by the other work then we will potentially get same encoding twice.

Reply:
`Dictionary<string, Encoding> AvailableEncodings` so this should be OK

Reply:
Thanks, @gerhardol, exactly that.